tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -405,7 +405,7 @@ public function testConnectionIsClosed()
     {
         $this->_conn->close();
 
-        $this->setExpectedException('Doctrine\\DBAL\\Exception\\DriverException');
+        //$this->setExpectedException('Doctrine\\DBAL\\Exception\\DriverException');
 
         $this->_conn->quoteIdentifier('Bug');
```

Comment:
Okay seems there are no other suggestions so I would accept using reflection here to do the assertion. @weaverryan can you do that?